All  enemy duelist monster card effects except Jester

ArdynXT opened the pull request:
Filled in new cards and fixed old ones as well

ArdynXT committed:
All  enemy duelist monster card effects except Jester

Filled in new cards and fixed old ones as well

Giratena commented on CSV/Language/English/Card/10053.txt:
Hmm what happened to this file? Does it have changes from the current translation PR in it?

ArdynXT replied:
I moved the files directly from my test copy which has a lot of the changes from my current translations and other peoples work. I was hoping that when merged to develop we can choose to only put  just the monster effects. I am not looking to push dialogue translations on the cards because these are also outdated.

ArdynXT replied:
I have removed all the dialogue and put the descriptions back in I must've tried to delete those to debug

Giratena commented on CSV/Language/English/Card/10054.txt:
This one here too!

Giratena commented on CSV/Language/English/Card/10055.txt:
And here

Giratena commented on CSV/Language/English/Card/10057.txt:
```diff
@@ -3,13 +3,9 @@ default
 EU57_paizuriAnime,
 EU57_bustShakeAnime,
 EU57_footShakeAnime,
-　道化の姿をしたモンスター。つかみどころのない態度であちこち動き回り、気が付けばこちらのそばにいてふところに潜り込んでいる。,
```
I think you accidentally removed this description here entirely!

Giratena commented on CSV/Language/English/Card/10058.txt:
Changes in this file here a shouldn't be added too.

Giratena commented on CSV/Language/English/Card/10059.txt:
```diff
@@ -3,12 +3,9 @@ default
 EU59_bustShowAnime,
 EU59_paizuriAnime,
 EU59_paizuriAuraAnime,
-　相手を魅了し、弱体化させることが得意な下級のサキュバス。魅了の呪いが込められたペンダントを、犬の首輪のように男にかけてくる。ペンダントを身に着けさせられた者は、首から上に常に快感が送られ、目の前にいるサキュバスたちの姿が通常よりもさらに美しく見えるようになり、抗うことができなくなってしまうのだという。,
```
This description was removed.

Giratena reviewed (changes requested):
Heyo @ArdynXT!

I just had a quick look over this PR and I can see some unusual stuff that aren't just monster card effects!

- Entirely seperate translations either from another PR or MTL 
- The removal of some enemy card descriptions, which we definitely wanna keep!

Maybe have a quick look over and clean it up a bit and then I can look at it :D

ArdynXT committed:
Addressed issues

Removed translations and added descriptions for jester and charm succubus

Giratena committed:
Merge branch 'develop' into Monster_effects_fix

Giratena commented on CSV/Language/English/Card/10002.txt:
```diff
@@ -1,6 +1,8 @@
 default
 サムライガール,Samurai Girl
 修行中の侍の少女。武技に優れるが夜伽も得意。相手の隙をついてふところに飛び込み、快楽や武力であっという間に相手を制圧してしまう。,A samurai girl in training; skilled in martial arts but also adept in various bedroom arts. She takes advantage of her opponent's openings to swiftly move in close; using both pleasure and martial prowess to quickly dominate them.
+也相當擅長侍寢。一旦抓到對手的空隙就算迅速鑽入，She's also quite skilled with pleasure attacks. Once she spots an opening in her opponent; she swiftly takes advantage of it.
```
What happened here? This looks like a Chinese line :O

ArdynXT replied:
I have been grabbing the Japanese from the Chinese translation files I must've grabbed the wrong line

ArdynXT replied:
The Chinese translation file is extremely strange I removed it and just made the English part of line 3

Giratena commented on CSV/Language/English/Card/10007.txt:
```diff
@@ -2,6 +2,9 @@ default,
 むちむちで張りのある体をした上級のスライム。胸で相手を虜にすることを得意とする。水質のきれいな湖に生息し、水辺に近づいてきた男性を人間に擬態して捕食する。,An advanced slime with a buxom and curvaceous body who is skilled at using her massive breasts to captivate men. She inhabits clean water lakes and often preys on men who approach the water's edge by mimicking the appearance of a human female.
+戦闘時「おっぱいに弱い」を付与する。,In battle; applies the “Weak to Breasts” status.
+.おっぱいに弱い相手から受けるダメージが半分になる。戦闘時「ぬるぬる」状態を付与する。,Damage taken from opponents weak to breasts is halved.
+In battle; applies the “Slippery” status.
```
This line has no Japanese text origin.

ArdynXT replied:
Found it! I took out the original line because it had parentheses and that caused it to not be replaced properly

Giratena commented on CSV/Language/English/Card/10007.txt:
```diff
@@ -2,6 +2,9 @@ default,
 EU7_paizuriAnime,
 むちむちで張りのある体をした上級のスライム。胸で相手を虜にすることを得意とする。水質のきれいな湖に生息し、水辺に近づいてきた男性を人間に擬態して捕食する。,An advanced slime with a buxom and curvaceous body who is skilled at using her massive breasts to captivate men. She inhabits clean water lakes and often preys on men who approach the water's edge by mimicking the appearance of a human female.
+戦闘時「おっぱいに弱い」を付与する。,In battle; applies the “Weak to Breasts” status.
+.おっぱいに弱い相手から受けるダメージが半分になる。戦闘時「ぬるぬる」状態を付与する。,Damage taken from opponents weak to breasts is halved.
```
The Japanese line here starts with a .

Was this intentional?

ArdynXT replied:
When looking at my chinese translation copy the period is there. The text doesn't show in my base copy so I believe I put that period in my chinese copy on accident and then copied it over.

Giratena commented on CSV/Language/English/Card/10018.txt:
```diff
@@ -3,10 +3,11 @@ default,
-（[1].戦闘時に「おっぱいに弱い」を付与する。[2].このカードは「おっぱいに弱い」相手から受ける戦闘ダメージが半分になる。）,([1]. Applies [Titty Fetish] to foe after combat. \n[2]. If foe has [Titty Fetish]. Reduces damage by 50% during combat.)
-さっきからわたしの大きな胸ばかり見ているな？気になるか？,Hey. You've been looking at my big breasts for a while now. What's the big idea? You like them?
-わたしは肉弾戦も得意だが、こういう夜の技も得意だ。ほら、胸で挟んで気持ちよくしてやろう。,While my specality is direct combat; I'm also good at nighttime arts such as this. Come on; l'll slip your dick between my boobs and make ya feel good.
+[1].相手ターン開始時に発動する。自分フィールド上のランダムなモンスター1体に、「戦闘ダメージ1割減」（1ターン）を付与する。\n,[1]. At the start of your opponent's turn; grant [90% Damage Reduction] to a random monster on your side of the field for 1 turn.
```
I believe this is actually 10% damage reduction now- which was fixed in an earlier PR.

Giratena commented on CSV/Language/English/Card/10018.txt:
```diff
@@ -3,10 +3,11 @@ default,
+[1].相手ターン開始時に発動する。自分フィールド上のランダムなモンスター1体に、「戦闘ダメージ1割減」（1ターン）を付与する。\n,[1]. At the start of your opponent's turn; grant [90% Damage Reduction] to a random monster on your side of the field for 1 turn.
+[1].戦闘時に「おっぱいに弱い」を付与する。,[1]. Applies [Titty Fetish] to foe after combat.
+[2].このカードは「おっぱいに弱い」相手から受ける戦闘ダメージが半分になる。,[2]. If foe has [Titty Fetish]. Reduces damage by 50% during combat.
+さっきからわたしの大きな胸ばかり見ているな？気になるか？,Hey. You've been looking at my big breasts for awhile now. What's the big idea? You like them?
```
These lines are missing their Japanese.

ArdynXT replied:
I don't quite follow on this one. I looked through the original file and the new one and it seems to be working properly in the game.

Giratena replied:
Hmm maybe whatever I saw got fixed somehow xD
I shall resolve this ^^

Giratena commented on CSV/Language/English/Card/10018.txt:
```diff
@@ -3,10 +3,11 @@ default,
+[1].戦闘時に「おっぱいに弱い」を付与する。,[1]. Applies [Titty Fetish] to foe after combat.
+[2].このカードは「おっぱいに弱い」相手から受ける戦闘ダメージが半分になる。,[2]. If foe has [Titty Fetish]. Reduces damage by 50% during combat.
+さっきからわたしの大きな胸ばかり見ているな？気になるか？,Hey. You've been looking at my big breasts for awhile now. What's the big idea? You like them?
+わたしは肉弾戦も得意だが、こういう夜の技も得意だ。ほら、胸で挟んで気持ちよくしてやろう。,While my specality is direct combat; I'm also good at nighttime arts such as this. Come on; l'll slip your dick between my boobs and make ya feel good.  
```
This one too.

ArdynXT replied:
Not sure on this one either let me know

Giratena commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,11 +1,10 @@
+ゾンビのモンスター娘。体はひんやりと冷たいが、胸やふとももは柔らかい。相手にゾンビウイルスを感染させるために口を使った誘惑をしてくるが、そうした本能とは別に、性的なことを通して「命」を強く感じられるため元々エッチなことが好き。,A zombie monster girl. Her body is cold to the touch; but her breasts and thighs are soft. She uses her mouth to seduce others and infect them with the zombie virus; but beyond that instinct; she genuinely enjoys sexual acts because they allow her to feel “life” intensely. Necro gave  ...
+口についている赤いものは血ではなくザクロの果汁である。肉を生で食べるのは不衛生だと言ってネクロが赤い果物を代わりに与えたところ、本人は気に入ってよくザクロをかじっている。,\n The red substance on her mouth not blood but pomegranate juice. Necro gave her the red fruit instead; saying eating raw meat was unsanitary; and she took to it; often munching on pomegranates.
+[1].このカードは「ゾンビ」状態を持ち。戦闘時に相手に「ゾンビ」状態を付与する。\n[2].このカードが墓地から特殊召喚されたとき発動する。フィールド上の「ゾンビ」状態のモンスターの数×100ポイント、自身の攻撃力をアップさせる。,"[1]. This card possesses the “Zombie” status. During combat; it inflicts the “Zombie” status on the opponent. \n[2]. When this card is Special Summoned from the Graveyard; activate this effect. Increase its ATK by 100 points for each “Zombie” monster on the field.
+[3].このカードが戦闘で「ゾンビ」状態の相手モンスターを破壊したとき発動する。破壊したモンスターのパラメーターを半分にして自分フィールド上に特殊召喚する。,"[3]. When this card destroys an opponent's monster in the “Zombie” state during battle; activate this effect. Special Summon that monster to your field with half its original parameters."
```
Quotes around this translation

Giratena commented on CSV/Language/English/Card/10053.txt:
```diff
@@ -1,11 +1,10 @@
-[2].このカードが墓地から特殊召喚されたとき発動する。フィールド上の「ゾンビ」状態のモンスターの数×100ポイント、自身の攻撃力をアップさせる。,
-[3].このカードが戦闘で「ゾンビ」状態の相手モンスターを破壊したとき発動する。破壊したモンスターのパラメーターを半分にして自分フィールド上に特殊召喚する。,
+ゾンビのモンスター娘。体はひんやりと冷たいが、胸やふとももは柔らかい。相手にゾンビウイルスを感染させるために口を使った誘惑をしてくるが、そうした本能とは別に、性的なことを通して「命」を強く感じられるため元々エッチなことが好き。,A zombie monster girl. Her body is cold to the touch; but her breasts and thighs are soft. She uses her mouth to seduce others and infect them with the zombie virus; but beyond that instinct; she genuinely enjoys sexual acts because they allow her to feel “life” intensely. Necro gave  ...
+口についている赤いものは血ではなくザクロの果汁である。肉を生で食べるのは不衛生だと言ってネクロが赤い果物を代わりに与えたところ、本人は気に入ってよくザクロをかじっている。,\n The red substance on her mouth not blood but pomegranate juice. Necro gave her the red fruit instead; saying eating raw meat was unsanitary; and she took to it; often munching on pomegranates.
```
->on her mouth [is] not blood but pomegranate juice

ArdynXT replied:
I am currently working on 10053 with a separate repo I believe I already addressed this in that branch I have fixed this issue here

Giratena commented on CSV/Language/English/Card/10054.txt:
```diff
@@ -57,8 +57,8 @@ EU54_footJobAnime,
 どうせ元々たいした脳みそじゃないんでしょ？\n自分からおちんちんシコシコしだしちゃうような、\n空っぽの脳みそは溶けちゃった方がいいよ❤,
-ほら、快楽の中でどろどろになろう？\n天使の息で全部溶かしてあげる❤\nふぅーー❤,
-あひぃぃ・・・・❤❤,
+e,
```
What happened here?

ArdynXT replied:
Must've been when I transferred them over to my ide from google sheets I'll fix that

Giratena commented on CSV/Language/English/Card/10054.txt:
```diff
@@ -57,8 +57,8 @@ EU54_footJobAnime,
-ほら、快楽の中でどろどろになろう？\n天使の息で全部溶かしてあげる❤\nふぅーー❤,
-あひぃぃ・・・・❤❤,
+e,
+=,
```
This line too

Giratena commented on CSV/Language/English/Card/10055.txt:
```diff
@@ -1,11 +1,10 @@
+死の世界を司る上級モンスター。肉体を離れた魂は彼女のもとに強く惹き寄せられ、決して逃れることはできない。\n　現世に未練がある魂は冥界に行くことを拒むことがあるが、魂を彼女に愛撫されるとその快感はとてつもないもので、すぐに未練を考えることもできなくなり、恍惚のなか彼女に言われるがまま冥界にわたってしまうのだという。,She is a high ranking monster in charge of the underworld. Souls that have left their bodies are strongly drawn to her and can never escape.
+[1].１ターンに１度、LPを500払い相手フィールド上の「ゾンビ」状態のモンスター１体を選択して発動できる。そのモンスターに2000ポイントのダメージを与える。相手フィールド上にモンスターが存在しない場合、この効果は相手プレイヤーを対象に発動することもできる。\n[2].相手が墓地からモンスターを特殊召喚したとき発動する。そのモンスターに2000ポイントのダメージを与える。,"[1].Once per turn; pay 500 LP and select a ""Zombie"" monster on your opponent's field. Deal 2000 points of damage to that monster. If there is no monster on your opponent's field; this effect can also target ...
+[3].このカードが手札に存在し相手フィールド上にアンデッドタイプのモンスターが2体以上存在する場合、自分フィールド上のモンスター１体をコストにして発動できる。このカードを手札から特殊召喚する。,[3]. If this card is in your hand and there are 2 or more Undead-type monsters on your opponent's field; you can sacrifice 1 monster on your field and special Summon this card from your hand.
+(ふとももコキで弱点攻撃時に拘束攻撃を行う。),(A binding attack is performed at the time of a weak attack with a futomokjoki.)
```
Lets translate/localise futomokjoki here.

Giratena commented on CSV/Language/English/Card/10055.txt:
```diff
@@ -1,11 +1,10 @@
-[3].このカードが手札に存在し相手フィールド上にアンデッドタイプのモンスターが2体以上存在する場合、自分フィールド上のモンスター１体をコストにして発動できる。このカードを手札から特殊召喚する。,
-(ふとももコキで弱点攻撃時に拘束攻撃を行う。),
+死の世界を司る上級モンスター。肉体を離れた魂は彼女のもとに強く惹き寄せられ、決して逃れることはできない。\n　現世に未練がある魂は冥界に行くことを拒むことがあるが、魂を彼女に愛撫されるとその快感はとてつもないもので、すぐに未練を考えることもできなくなり、恍惚のなか彼女に言われるがまま冥界にわたってしまうのだという。,She is a high ranking monster in charge of the underworld. Souls that have left their bodies are strongly drawn to her and can never escape.
+[1].１ターンに１度、LPを500払い相手フィールド上の「ゾンビ」状態のモンスター１体を選択して発動できる。そのモンスターに2000ポイントのダメージを与える。相手フィールド上にモンスターが存在しない場合、この効果は相手プレイヤーを対象に発動することもできる。\n[2].相手が墓地からモンスターを特殊召喚したとき発動する。そのモンスターに2000ポイントのダメージを与える。,"[1].Once per turn; pay 500 LP and select a ""Zombie"" monster on your opponent's field. Deal 2000 points of damage to that monster. If there is no monster on your opponent's field; this effect can also target ...
```
This line is wrapped in quotes

Giratena commented on CSV/Language/English/Card/10059.txt:
```diff
@@ -3,12 +3,10 @@ default
+　相手を魅了し、弱体化させることが得意な下級のサキュバス。魅了の呪いが込められたペンダントを、犬の首輪のように男にかけてくる。ペンダントを身に着けさせられた者は、首から上に常に快感が送られ、目の前にいるサキュバスたちの姿が通常よりもさらに美しく見えるようになり、抗うことができなくなってしまうのだという。,　A low-level succubus skilled at enchanting and weakening her prey. She fastens a pendant imbued with magic around a man's neck like a dog collar. Those wearing the pendant constantly feel pleasure coursing through their upper body; and the succu ...
+[1].このカードが場に出たとき発動する。デッキから「チャームペンダント」を１枚手札に加える。\n[2].1ターンに１度、相手フィールド上の「魅了」状態のキャラクターを選択して発動できる。そのキャラクターに「早漏化」を付与する。\n[3].相手の「魅了」状態のモンスターが破壊されたとき発動する。カードを1枚ドローする。,[1].When this card is put onto the field; you can activate this effect. Add 1 "{%%60023}" from your Deck to your hand. \n[2].Once per turn; you can activate this effect by selecting a character on your opponent's field that is in the "Cha ...
+[4].このカードが存在する限り、相手は「魅了」状態のモンスターを上級召喚のコストにすることはできない。,[4].As long as this card is in play; your opponent cannot use monsters in the "Charm" state as costs for higher level Summons.
+戦闘時に弱点を付与する。「早漏化」を付与した相手の現在HPが自身の攻撃力の2倍以下の場合、オナニー誘惑を行う。,Inflicts a weakness during combat. If the target's current HP is 2 times or less than your attack power while under the effect of “Premature Ejaculation” it will attempt to seduce them into masturbation.
```
This last line is a little weird. Try simple English. Something like:

"This card will try to seduce the target into masturbating."/"This card will try to make the target masturbate."

ArdynXT replied:
Made changes here feel free to take a look!

Giratena commented on CSV/Language/English/Card/10057.txt:
```diff
@@ -3,13 +3,10 @@ default
+　道化の姿をしたモンスター。つかみどころのない態度であちこち動き回り、気が付けばこちらのそばにいてふところに潜り込んでいる。,　A monster in the guise of a clown. It moves about with an elusive demeanor; darting here and there; and before you know it; she is right beside you; slipping into your pocket.
+[1].このカードは「隠密」「弱点付与」を持つ。\n[2].このカードが通常召喚に成功したとき、LPを800払って発動する。相手フィールド上の最もレベルが低いモンスター（このカードと同名のカードは除く）とこのカードのコントロールを入れ替える。その後、そのモンスターに「攻撃不能（1ターン）」を付与する。\n[3].このカードのコントローラーが元々の持ち主と違う場合、このカードは相手キャラクターに攻撃できず、自分キャラクターに対してのみ攻撃できる。このときこのカードは反撃されない。特性：「潛行」「弱點賦予」。,[1].This card has Stealth and Weakness Assignment \n[2]. When this card successfully Normal Summons pay 800 LP to activate. Swap control betwee ...
+[4].このカードのコントローラーが元々の持ち主と違う場合、自分ターン終了時に発動する。相手はカードを１枚ドローする。,[3].If this card's controller is different from its original owner; activate this effect at the end of your turn. Your opponent draws one card.
+戦闘時に弱点を付与する。,Inflicts a Weakness during battle.
```
"A weakness"? Maybe just "weakness".

ArdynXT replied:
Ah yeah missed this thank you!

Giratena commented on CSV/Language/English/Card/10058.txt:
```diff
@@ -1,6 +1,9 @@
 EU58_bustZoomAnime,
+[1].１ターンに１度、相手キャラクターを選択して発動できる。そのキャラクターに「おっぱいに弱い」を付与する。すでに「おっぱいに弱い」状態だった場合、「魅了」を付与する。すでに「魅了」状態だった場合、「早漏化」を付与する。,[1]. Once per turn; you can select an opponent's character to activate this effect. Apply “Weak to Breasts” to that character. If the character already has “Weak to Breasts” apply “Charm” instead. If the target is already Charmed; apply Premature Ejaculation.
+[2].このカードは「おっぱいに弱い」相手から戦闘・効果ダメージを受けない。,[2]. This card takes no combat or effect damage from opponents with “Weak to Breasts”. During combat; apply “Weak to Breasts”.
+戦闘時に「おっぱいに弱い」を付与する。「早漏化」を付与した相手の現在HPが自身の攻撃力の2倍以下の場合、オナニー誘惑を行う。,If an opponent with Premature Ejaculation has current HP equal to or less than twice your ATK; perform Masturbation Temptation.
```
Try using simple English here. This one is quite stiff.

Giratena replied:
@ArdynXT just this comment left!

Giratena commented on CSV/Language/English/Card/10058.txt:
```diff
@@ -1,6 +1,9 @@
+default,
 ロリ巨乳サキュバス,Shortstack Succubus
 EU58_bustZoomAnime,
+[1].１ターンに１度、相手キャラクターを選択して発動できる。そのキャラクターに「おっぱいに弱い」を付与する。すでに「おっぱいに弱い」状態だった場合、「魅了」を付与する。すでに「魅了」状態だった場合、「早漏化」を付与する。,[1]. Once per turn; you can select an opponent's character to activate this effect. Apply “Weak to Breasts” to that character. If the character already has “Weak to Breasts” apply “Charm” instead. If the target is already Charmed; apply Premature Ejaculation.
```
Watch your tenses with that last sentence. Keep it consistent with what you would normally see on a Yugioh/Mtg card!

Giratena commented on CSV/Language/English/Card/10070.txt:
```diff
@@ -5,10 +5,8 @@ EU70_handJobWithKissAnime,
-(ハード以上の場合、さらにデッキからトラップカードをランダムに１枚手札に加える),
-[3].このカードが破壊され墓地へ送られたとき発動する。「怪盗の挑戦状」を1枚生成し手札に加える。,
+[1].このカードは「隠密」「ステルス」を持つ。このカードは相手プレイヤーに直接攻撃できる。\n[2].このカードが相手プレイヤーに戦闘ダメージを与えたとき発動する。与えたダメージの分だけ自分プレイヤーのLPを回復する。,[1]. This card has “Stealth” and “Hidden”. This card can attack your opponent directly. \n[2]. When this card inflicts battle damage to your opponent; activate this effect. Recover LP equal to the damage inflicted.
+(ハード以上の場合、さらにデッキからトラップカードをランダムに１枚手札に加える)\n[3].このカードが破壊され墓地へ送られたとき発動する。「怪盗の挑戦状」を1枚生成し手札に加える。,(If Hard difficulty or higher; add 1 Trap Card from your Deck to your hand at random) \n[3]. When this card is destroyed and sent to the Graveyard; activate this effect. Generate 1 “Thief's Challenge” and add it to your hand.
```
Is Thief's Challenge the current translation of that card?

ArdynXT replied:
That's 50096 I could add it as the identifier the current translation is Phantom Thief's challenge.
It doesn't look like the Japanese translation makes reference to that so I wouldn't want to break it.

Giratena replied:
Ok in that case let's leave this as is. I'll resolve ^^

Giratena commented on CSV/Language/English/Card/10081.txt:
All these changes are too big and should be in an entirely separate PR! Though what happened here anyway?

ArdynXT replied:
Sorry a lot of these are from my test copy which I put all the different branches into just to fill out a lot of the translation gaps I will remove these and all the other ones shortly

Giratena commented on CSV/Language/English/Card/10083.txt:
These changes should also be in their own PR ^^

Giratena commented on CSV/Language/English/Card/10082.txt:
I think you overwrote/corrupted this file a bit?

Giratena reviewed (changes requested):
Reviewed!

Hmm there is still a little bit of clean up to do here, but I was at least able to look through it all this time.

I suspect some files were edited without pulling first, as I can see some descriptions were already overwritten, but I *think* it's OK.

Also those entirely separate card proofreading should not be in here and should be in their own PR so they can be checked properly ^^

ArdynXT committed:
Fixed or attempted to fix all known issues!

ArdynXT committed:
Merge branch 'Monster_effects_fix' of https://github.com/ArdynXT/Succubus-Duel-English-Translation into Monster_effects_fix

Giratena commented on CSV/Language/English/Card/10059.txt:
```diff
@@ -6,7 +6,7 @@ EU59_paizuriAuraAnime,
 [1].このカードが場に出たとき発動する。デッキから「チャームペンダント」を１枚手札に加える。\n[2].1ターンに１度、相手フィールド上の「魅了」状態のキャラクターを選択して発動できる。そのキャラクターに「早漏化」を付与する。\n[3].相手の「魅了」状態のモンスターが破壊されたとき発動する。カードを1枚ドローする。,[1].When this card is put onto the field; you can activate this effect. Add 1 "{%%60023}" from your Deck to your hand. \n[2].Once per turn; you can activate this effect by selecting a character on your opponent's field that is in the "Cha ...
 [4].このカードが存在する限り、相手は「魅了」状態のモンスターを上級召喚のコストにすることはできない。,[4].As long as this card is in play; your opponent cannot use monsters in the "Charm" state as costs for higher level Summons.
-戦闘時に弱点を付与する。「早漏化」を付与した相手の現在HPが自身の攻撃力の2倍以下の場合、オナニー誘惑を行う。,Inflicts a weakness during combat. If the target's current HP is 2 times or less than your attack power while under the effect of “Premature Ejaculation” it will attempt to seduce them into masturbation.
+戦闘時に弱点を付与する。「早漏化」を付与した相手の現在HPが自身の攻撃力の2倍以下の場合、オナニー誘惑を行う。,Inflicts weakness during combat. If the target's current HP is 2 times or less than your attack power while under the effect of “Premature Ejaculation” this card will try seducing the target monster into masturbating.
```
"this card will try **to seduce** the target monster into masturbating"

Giratena commented on CSV/Language/English/Card/10007.txt:
```diff
@@ -3,8 +3,8 @@ default,
-.おっぱいに弱い相手から受けるダメージが半分になる。戦闘時「ぬるぬる」状態を付与する。,Damage taken from opponents weak to breasts is halved.
-In battle; applies the “Slippery” status.
+おっぱいに弱い相手から受けるダメージが半分になる。戦闘時「ぬるぬる」状態を付与する。,Damage taken from opponents weak to breasts is halved.
+時賦予對方「黏稠」狀態。,In battle; applies the “Slippery” status.
```
Hmm this is Chinese text. What's doing on here?

ArdynXT replied:
I believe I grabbed this from the next line of the chinese copy. I think the lines are together so I am going to remove the chinese and have them in the same sentence hopefully that fixes this issue

Giratena commented on CSV/Language/English/Card/10055.txt:
```diff
@@ -2,9 +2,9 @@ default,
+[1].１ターンに１度、LPを500払い相手フィールド上の「ゾンビ」状態のモンスター１体を選択して発動できる。そのモンスターに2000ポイントのダメージを与える。相手フィールド上にモンスターが存在しない場合、この効果は相手プレイヤーを対象に発動することもできる。\n[2].相手が墓地からモンスターを特殊召喚したとき発動する。そのモンスターに2000ポイントのダメージを与える。,[1].Once per turn; pay 500 LP and select a "Zombie" monster on your opponent's field. Deal 2000 points of damage to that monster. If there is no monster on your opponent's field; this effect can also target an ...
 [3].このカードが手札に存在し相手フィールド上にアンデッドタイプのモンスターが2体以上存在する場合、自分フィールド上のモンスター１体をコストにして発動できる。このカードを手札から特殊召喚する。,[3]. If this card is in your hand and there are 2 or more Undead-type monsters on your opponent's field; you can sacrifice 1 monster on your field and special Summon this card from your hand.
-(ふとももコキで弱点攻撃時に拘束攻撃を行う。),(A binding attack is performed at the time of a weak attack with a futomokjoki.)
+(ふとももコキで弱点攻撃時に拘束攻撃を行う。),(A binding attack is performed at the time of a weak attack with a Thigh squeeze.)
```
Capital T should be lowercase here.

Giratena commented on CSV/Language/English/Card/10081.txt:
```diff
@@ -1,278 +1,278 @@
 default,,
-魅惑のマーメイド,Seductive Mermaid,
+魅惑のマーメイド,,Seductive Mermaid,,
```
Double commas here will break the translation.

ArdynXT replied:
This is due to my excel editor I will remove them

Giratena commented on CSV/Language/English/Card/10081.txt:
This file has double commas which will break the translation.

Giratena commented on CSV/Language/English/Card/10082.txt:
This file removes all the existing translations.

Giratena commented on CSV/Language/English/Card/10083.txt:
This file also destroys some translations.

Giratena reviewed (changes requested):
Hello again @ArdynXT!

I checked out the changes. Looks like you've accidentally removed some translations in here ^^;

I also added more comments for other issues I saw. Please take a look!

There's also a merge conflict you'll need to resolve ^^

ArdynXT committed:
Resolved Conflicts

I hadn't checked in on the develop branch so I was unsure if the mermaids were in develop. I simply removed them. Besides the monster card descriptions everything should be 1to1 with develop

Giratena commented:
@ArdynXT this Branch still has conflicts just FYI.

ArdynXT committed:
Merge branch 'develop' into Monster_effects_fix

ArdynXT committed:
1058 and 1059 conflicts fix

ArdynXT committed:
Merge branch 'Monster_effects_fix' of https://github.com/ArdynXT/Succubus-Duel-English-Translation into Monster_effects_fix

ArdynXT committed:
10063 and 10082 fix

Issues after merge this should be everything now!

Giratena commented:
@ArdynXT you've still got a merge conflict here!

Giratena reviewed (changes requested):
Your PR is conflicting with all the PRs we've been merging lately.

You'll need to fix these up, and we can chalk all this up to learning experience to not do big sweeping changes across files that might change a lot in future ^^

Giratena committed:
Merge branch 'develop' into Monster_effects_fix

Giratena committed:
Update 10002.txt

Giratena committed:
Update 10005.txt

Giratena commented:
Closing due as this PR has now become quite stale, and I think the translation has progressed past the point for this to add anything.
